test/multiverse/suites/concurrent_ruby/config/newrelic.yml

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  error_collector:
+    enabled: true
+  apdex_t: 0.5
+  monitor_mode: true
+  license_key: bootstrap_newrelic_admin_license_key_000
+  instrumentation:
+    concurrent_ruby: <%= $instrumentation_method %>
```

Comment:
The tests pass both ways, but if you'd like to see what happens when Thread instrumentation is on, try adding `thread.tracing: true` on the next line.